data_policy.md

```diff
@@ -1,17 +1,21 @@
 # ISIS Data Policy  
 
-## ISIS data management policy
+The ISIS data policy lays out the guarantees ISIS provides, and the requirements for users of the facilities, pertaining to the curation, access, ownership, use and storage of data collected at ISIS.
```

Comment:
```suggestion
The ISIS data policy lays out the guarantees ISIS provides, and the requirements for users of the facility, pertaining to the curation, access, ownership, use and storage of data collected at ISIS.
```

Reply:
changed facilities to facility since most of our users may read facilities as multiple neutron and muon facilities